addons/nfs-provisioner/resources/04-pvc.yml.erb

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
So this makes the assumption that there's already an existing default class setup for the cluster. Just thinking that can we really make that assumption?